src/OneExplorer.ts

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as fs from 'fs';
+import * as path from 'path';
+import * as vscode from 'vscode';
+
+enum NodeType{
+  directory,
+  model,
+  config
+}
+
+interface Node {
+  type: NodeType, name: string, childs: Node[], uri: vscode.Uri
+}
+
+export class OneNode extends vscode.TreeItem {
+  constructor(
+      public readonly label: string,
+      public readonly collapsibleState: vscode.TreeItemCollapsibleState,
+      public readonly node: Node,
+  ) {
+    super(label, collapsibleState);
+
+    this.tooltip = `${this.label}`;
```

Comment:
How about showing full file path like file explorer view?

_Originally posted by @jyoungyun in https://github.com/Samsung/ONE-vscode/pull/495#discussion_r858158508_

Reply:
You can use the fsPath(?) of node's Uri. 
```suggestion
    this.tooltip = `${this.node.uri.fsPath}`;
```

Reply:
Done

Reply:
Why don't you add `TODO` at line 38?

Reply:
This TODOs in comments are all resolved.